client/signup/steps/design-type-with-atomic-store/index.jsx

```diff
@@ -172,12 +161,12 @@ class DesignTypeWithAtomicStoreStep extends Component {
 			'is-hidden': ! this.state.showStore,
 		} );
 
-		const designTypeListClassName = classNames( 'design-type-with-atomic-store__list', {
+		const designTypeListClassName = classNames( 'design-type-with-store__list', {
 			'is-hidden': this.state.showStore,
 		} );
 
 		return (
-			<div className="design-type-with-atomic-store__substep-wrapper">
+			<div className="design-type-with-atomic-store__substep-wrapper design-type-with-store__substep-wrapper">
```

Comment:
Both classNames are needed for namespace lint rule?

Reply:
The first one is necessary for the lint rule, the second one - for making the container relative. Can be cleaned up after we migrate the rest of the steps to use `TileGrid`.